.eslintignore

```diff
@@ -0,0 +1 @@
+/src/tests
```

Comment:
tests should still be covered though right?

Reply:
ESLint goes crazy when it sees we are using Jest functions without importing it... that's why I had to ignore it

Reply:
Was there no way to declare it as a global?

Reply:
I looked into this and found that we can remove the `.eslintignore` file, so we get linting in our tests. 
To add eslint support for jest all we need to do is update the `eslintrc.js` file's `env` property like so:
```js
module.exports = {
env: {
  //..
  jest: true
},
//..
```

[ref](https://stackoverflow.com/questions/31629389/how-to-use-eslint-with-jest)

Reply:
Ok... updating the PR!